charts/osm/values.schema.json

```diff
@@ -244,12 +244,14 @@
                     "examples": [
                         {
                             "registry": "openservicemesh",
+                            "name": "osm-controller",
```

Comment:
If `name` is supposed to map to an object then this doesn't seem valid. Given I'm not aware of anything parsing these examples, I'd say it's not worth adding any more to them here.